### ISSUE TITLE
Enabling/disabling a mod seems to create a new file rather than move the existing file

### ISSUE THREAD

**Issue by alphaniner** [ATLauncher/ATLauncher#760]
### What happened?

1. Install a mod to an instance.
2. Query the File ID of the mod:
```
> fsutil file queryfileid .\instances\Forge2\mods\jei-1.19.2-forge-11.6.0.1015.jar
File ID is 0x0000000000000000000c000000016a6d
```
3. (Optional) Move the file around to confirm that move operations shouldn't change FileID:
```
> move .\instances\Forge2\mods\jei-1.19.2-forge-11.6.0.1015.jar .\instances\Forge2\disabledmods
> fsutil file queryfileid .\instances\Forge2\disabledmods\jei-1.19.2-forge-11.6.0.1015.jar
File ID is 0x0000000000000000000c000000016a6d
> move .\instances\Forge2\disabledmods\jei-1.19.2-forge-11.6.0.1015.jar .\instances\Forge2\mods
> fsutil file queryfileid .\instances\Forge2\mods\jei-1.19.2-forge-1.6.0.1015.jar
File ID is 0x0000000000000000000c000000016a6d
```
4. Disable the mod through ATLauncher.
5. See that the File ID has changed, indicating it is actually a new file:
```
> fsutil file queryfileid ATLauncher\instances\Forge2\disabledmods\jei-1.19.2-forge-11.6.0.1015.jar
File ID is 0x0000000000000000000d000000016a82
```
6. Re-enable the mod through ATLauncher.
7. See that the File ID has changed again.

Looking at the timestamps is probably enough really, but this is more definitive.

The expectation, of course, is that enabling or disabling a mod would work like a move operation and not create a new file.

### Operating System

Windows

### Relevant log output

```shell
N/A
```

### Additional Information

The real reason I discovered this is that I was experimenting with using hardlinks to avoid duplication between similar instances (and different loaders). When I disabled a mod, I found it was no longer linked to the "primary" copy.

Enabling & disabling mods is probably not something people do often, so I don't expect this is a big deal. And if there's a good reason for the existing behaviour, so be it. It just seems like it should be possible to enable/disable without creating a new file.

### Is this actually a launcher bug?

- [X] I am sure this is a bug with the launcher and not an issue with running Minecraft

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by RyanTheAllmighty** [comment]
Yeah all our "move" operations are just copy operations under the hood. No idea why, they were all mostly written 10+ years ago now.

Makes more sense to actually move files. Will fix up as part of the current work in #556 which is removing the disabledmods folder for preference of a `.disabled` extension already, so can add it in with those changes

**Comment by alphaniner** [comment]
Cool, thanks for the heads up.

 I prefer the `.disabled` extension method too, so that's a bonus.
